include/fuzzy/ngram_matches.hh

```diff
@@ -37,6 +34,7 @@ namespace fuzzy
     AgendaItem* get_agendaitem(unsigned);
     AgendaItem* new_agendaitem(unsigned, unsigned);
     unsigned _p_length;
+    unsigned min_seq_len;
```

Comment:
Do we want to prefix the attribute with an underscore?

Reply:
Done